### ISSUE TITLE
No longer able to send messages

### ISSUE THREAD

**Issue by alfwro13** [haberda/signal-addon#120]
Addon stopped working a couple of hours ago. I am no longer able to send messages. There are no error messages in the logs but when I try to send a message using Home Assistant's Developer tools/Services it complains: Failed to call service notify.signal_ha_test. Unknown error

**Comment by reuhno** [comment]
Me too since this evening

**Comment by alfwro13** [comment]
I think this might be related to: https://github.com/AsamK/signal-cli/issues/1439

**Comment by reuhno** [comment]
Yes, I saw and i think too. Hope for a release soon.  🤞🤞🤞

**Comment by haberda** [comment]
Yeah, as soon as there is an upstream release I will issue an update.

**Comment by alfwro13** [comment]
haberda - thank you for your support, it would appear that the upstream is now fixed

**Comment by kojanovski** [comment]
Hi,

In my case it stopped working after the installation HA Release 2024.1.6. I get a response as below:

[core-ssh ~]$ curl -X GET -H "Content-Type: application/json" 'http://192.168.1.131:8080/v1/identities/+xxxxxxxxxxx'
{"error":"Error while checking account +xxxxxxxxxxx: signal-cli version is too old for the Signal-Server, please update.\n"}[core-ssh ~]$

**Comment by steveredden** [comment]
> haberda - thank you for your support, it would appear that the upstream is now fixed

there's multiple upstreams involved - namely this one got released an hour ago:

https://github.com/bbernhard/signal-cli-rest-api/releases/tag/0.81

**Comment by derjoerg** [comment]
Looks good to me.

Thanks a lot!

**Comment by alfwro13** [comment]
perfect, all working fine now

**Comment by kojanovski** [comment]
The fix works fine. Many thanks,
